src/soter/boringssl/soter_ecdsa_common.h

```diff
@@ -14,16 +14,15 @@
 * limitations under the License.
 */
 
-#ifndef SOTER_EC_COMMON_H
-#define SOTER_EC_COMMON_H
+#ifndef SOTER_BORINGSSL_ECDSA_COMMON_H
+#define SOTER_BORINGSSL_ECDSA_COMMON_H
 
-#include <soter/soter.h>
 #include <soter/soter_ec_key.h>
-#include "soter_engine.h"
+#include <soter/soter_error.h>
+#include <soter/boringssl/soter_engine.h>
```

Comment:
why we use angle braces here? should boringssl/soter_engine.h be public?

Reply:
Mmm... I think you're right, it's not public. I initially assumed _all_ headers to be public and thus use only angled brackets there (and quotes in *.c files). I'll review the headers and will tweak the quotes.

Now when I think of it... It's kinda hard to distinguish public headers from private ones. I've seen quite a few libraries put their public headers into a separate top-level directory `include`. For example:

```
include
  themis
    secure_cell.h
    themis.h
    ...
src
  themis
    message.h
    portable_endian.h
    secure_session_utils.h
    ...
```

How do you feel about moving to this layout in the next PR?

Reply:
Based on the include analysis, here are the public headers. I.e., this gets included with `#include <themis/themis.h>`:

```
soter/soter.h
soter/soter_asym_cipher.h
soter/soter_asym_ka.h
soter/soter_asym_sign.h
soter/soter_error.h
soter/soter_hash.h
soter/soter_hmac.h
soter/soter_kdf.h
soter/soter_rand.h
soter/soter_rsa_key.h
soter/soter_rsa_key_pair_gen.h
soter/soter_sym.h

themis/themis.h
themis/themis_error.h
themis/secure_cell.h
themis/secure_comparator.h
themis/secure_keygen.h
themis/secure_message.h
themis/secure_session.h
```

And these ones are private:

```
soter/boringssl/*
soter/ed25519/*
soter/openssl/*
soter/soter_container.h
soter/soter_crc32.h
soter/soter_ec_key.h
soter/soter_sign_ecdsa.h
soter/soter_sign_rsa.h
soter/soter_t.h

themis/message.h
themis/portable_endian.h
themis/secure_cell_alg.h
themis/secure_comparator_t.h
themis/secure_message_wrapper.h
themis/secure_session_peer.h
themis/secure_session_t.h
themis/secure_session_utils.h
themis/sym_enc_message.h
```